### ISSUE TITLE
Search results\See more link injects html improperly

### ISSUE THREAD

**Issue by Blackbaud-TomRamsey** [blackbaud/skyux-lib-help#22]
When viewing search results in the help panel, clicking 'see more' causes the additional results to be injected into the DOM as an escaped string rather then actual DOM elements.

https://files.slack.com/files-tmb/T02BBN218-F7J48ATQD-3ad69ab039/pasted_image_at_2017_10_13_11_10_am_480.png

**Comment by Blackbaud-BrandonJones** [comment]
Fixed with this PR: https://github.com/blackbaud/bb-help/pull/113  will release and merge once BB-Con is complete.

**Comment by Blackbaud-BrynaGleich** [comment]
Should the See More link be blue to show that it is a hyperlink? Currently it shows as plain black.

**Comment by Blackbaud-BrandonJones** [comment]
I don't believe this has any styles attributed to it or style requirements. I can add another task to investigate trying out a blue link instead.

**Comment by Blackbaud-BrynaGleich** [comment]
Please -- when you have the time for it (not critical need, but solid nice-to-have) All other clickable links in the product and panel have a color to them, except this one.
